### PR TITLE
Deprecate the range function

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: doc/functions/range.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ``range``
 =========
 
+.. warning::
+
+    The ``attribute`` filter is deprecated as of Twig 3.15. Use the ``..``
```
Comment (xabbuh) [inline doc/functions/range.rst]:
```suggestion
    The ``range()`` function is deprecated as of Twig 3.15. Use the ``..``
```

Comment (fabpot) [reply doc/functions/range.rst]: oops :)

File: doc/functions/range.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 ``range``
 =========
 
+.. warning::
+
+    The ``range`` filter is deprecated as of Twig 3.15. Use the ``..`` operator
```
Comment (xabbuh) [inline doc/functions/range.rst]:
```suggestion
    The ``range`` function is deprecated as of Twig 3.15. Use the ``..`` operator
```

:)

Comment (fabpot) [reply doc/functions/range.rst]: 😂

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Reverted, I didn't realize that `range` has one additional feature that `..` does not have: step.

**Pull request merged**